tests/func-tests/monitoring_test.go

```diff
@@ -39,7 +39,7 @@ const (
 	criticalImpact
 )
 
-var _ = Describe("[crit:high][vendor:cnv-qe@redhat.com][level:system]Monitoring", Serial, Ordered, Label(tests.OpenshiftLabel), func() {
+var _ = Describe("[crit:high][vendor:cnv-qe@redhat.com][level:system]Monitoring", Serial, Ordered, Label(tests.OpenshiftLabel, "monitoring"), func() {
```

Comment:
if we start using this for more tests, can we replace `"monitoring"` with something like `tests.MonitoringLabel` or so?

Reply:
Do we have additional monitoring tests? if yes, I'll do it in a followup PR.

Reply:
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/015e6f76e2f95faafabb532811bdc85aa4fa10c0/tests/func-tests/observability_controller_test.go#L16 could belong to it I think